apps/ExampleApp/package.json

```diff
@@ -26,15 +26,16 @@
     "web": "npx expo start --web",
     "bundle:web": "npx expo export --platform web",
     "serve:web": "npx server dist",
-    "prebuild:clean": "npx expo prebuild --clean"
+    "prebuild:clean": "npx expo prebuild --clean",
+    "eas-build-post-install": "yarn -T run ci:build"
   },
   "dependencies": {
     "@expo-google-fonts/space-grotesk": "^0.2.2",
     "@expo/metro-runtime": "~3.2.1",
-    "@infinitered/react-native-mlkit-document-scanner": "^1.0.0",
-    "@infinitered/react-native-mlkit-face-detection": "^1.0.0",
-    "@infinitered/react-native-mlkit-image-labeling": "^1.0.0",
-    "@infinitered/react-native-mlkit-object-detection": "^1.0.0",
+    "@infinitered/react-native-mlkit-document-scanner": "workspace:*",
+    "@infinitered/react-native-mlkit-face-detection": "workspace:*",
```

Comment:
does this interfere with changeset?

Reply:
i'm not sure, @trevor-coleman, thoughts?

Reply:
How does changeset work with `workspace:^1.0.0`?

Reply:
The reason why I recommended to use `workspace:...` ([workspace protocol](https://yarnpkg.com/protocol/workspace)) is because it's the best balance between visibility, maintainability, and making sure monorepos work.

- If you use just the version itself, e.g. `^1.0.0`, it might download the package instead of using the package from workspace when the version ranges get out of date. This is less of an issue if you use changeset to manage these versions for you (which I didn't know)
- @mazenchami and I were also talking about manually linking the packages through `expo.autolinking` definition in the **package.json**. But, this might cause the JS and native side to go out of sync, when yarn decides to download the version from registry (while the native side still gets linked from the workspace)
- The `workspace` protocol is a helpful tool to avoid all of that, and just instruct yarn to always use the workspace version (it can have a version constraint, which will result in an error if the versions cant be satisfied from the monorepo -- better visibility when this happens)

But feel free to change it however you see fit ofc 😄

Reply:
Looks like Changesets does support the workspace protocol ([issue](https://github.com/changesets/changesets/issues/307), [PR](https://github.com/changesets/changesets/pull/311)), so lets got with `workspace:^2.0.0` for the versions and we'll get the best of both worlds.

Reply:
fix here: https://github.com/infinitered/react-native-mlkit/pull/154/commits/39b12c0607766bc4b2c45a5927274188b76ff6b6